posix/subsystem/src/cgroupfs.hpp

```diff
@@ -109,14 +109,27 @@ struct RegularNode : FsNode, std::enable_shared_from_this<RegularNode> {
 
 struct SuperBlock final : FsSuperblock {
 public:
-	SuperBlock() = default;
+	SuperBlock() {
+		mountMinor_ = getAnonymousMountIdAllocator().allocate();
+	}
 
 	FutureMaybe<std::shared_ptr<FsNode>> createRegular(Process *) override;
 	FutureMaybe<std::shared_ptr<FsNode>> createSocket() override;
 
 	async::result<frg::expected<Error, std::shared_ptr<FsLink>>>
 			rename(FsLink *source, FsNode *directory, std::string name) override;
 	async::result<frg::expected<Error, FsFileStats>> getFsstats() override;
+
+	std::string getFsType() override {
+		return "cgroup2";
+	}
+
+	dev_t deviceNumber() override {
+		return makedev(0, mountMinor_);
+	}
+
+private:
+	unsigned int mountMinor_;
```

Comment:
This has nothing to do with mounts, hence:
```suggestion
	unsigned int deviceMinor_;
```
Likewise for the other file systems.